src/google/protobuf/compiler/objectivec/import_writer.cc

```diff
@@ -166,6 +162,25 @@ void ImportWriter::AddRuntimeImport(const std::string& header_name) {
   protobuf_imports_.push_back(header_name);
 }
 
+std::string ImportWriter::ModuleForFile(const FileDescriptor* file) {
+  if (!file) {
+    return "";
+  }
+
```

Comment:
ABSL_DCHECK(!IsProtobufLibraryBundledProtoFile(file));

for safety sake?

Reply:
done